ui/ui-frontend/projects/archive-search/src/app/archive/filing-holding-scheme/leaves-tree/leaves-tree.component.ts

```diff
@@ -261,7 +261,7 @@ export class LeavesTreeComponent implements OnInit, OnChanges, OnDestroy {
       this.archiveSharedDataService.selectedUnit$.subscribe((selectedUnit: Unit) => {
         if (selectedUnit) {
           this.unitId = selectedUnit['#id'];
-          this.allunitups = selectedUnit['#allunitups'];
+          this.allunitups = selectedUnit['#allunitups'] || [];
```

Comment:
Dans [une autre MR](https://github.com/ProgrammeVitam/vitam-ui/pull/2541/files#diff-6a44bcec3f45591a015ed0324b6f35bbb6dbc51573b610ab3dca4e33376fe4b9), on a : 
```suggestion
          this.allunitups = selectedUnit['#allunitups'] ? selectedUnit['#allunitups'] : [];
```